strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/users/PasswordEncoderController.java

```diff
@@ -0,0 +1,38 @@
+package org.carlspring.strongbox.controllers.users;
+
+import javax.inject.Inject;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+/**
+ * @author Przemyslaw Fusik
+ */
+@Controller
+@RequestMapping("/passwords")
+@Api(value = "/passwords")
```

Comment:
Could the request mapping for this password encoder be something like `/users/password-encoder`, or something along these lines? I mean... later on we'll otherwise have to tidy up again? What do you think?